app/src/main/java/org/oppia/app/walkthrough/end/WalkthroughFinalFragment.kt

```diff
@@ -1,4 +1,4 @@
-package org.oppia.app.walkthrough
+package org.oppia.app.walkthrough.end
```

Comment:
package name for this should `final` instead of `end` to make it consistent with internal files.

Reply:
apparently final cannot be used as a package name

Reply:
Ohh got it, keyword issue. Thanks.